package.json

```
@@ -7,7 +7,9 @@
     "ios": "react-native run-ios",
     "start": "react-native start",
     "test": "jest",
-    "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
+    "prepare": "husky install",
+    "format": "prettier \"src/**/*.{js,ts,jsx,tsx}\""
```

To actually write the formatting changes required:

```suggestion
    "format": "prettier \"src/**/*.{js,ts,jsx,tsx}\" --write"
```